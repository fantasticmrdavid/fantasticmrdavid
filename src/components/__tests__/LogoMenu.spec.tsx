import React from 'react';
import { render } from "@testing-library/react";
import Ui from 'components/LogoMenu/ui';

describe('Component -> LogoMenu', () => {
  it('should render the LogoMenu UI without throwing an error', () => {
    expect(render(<Ui />)).toMatchSnapshot();
  });
});
