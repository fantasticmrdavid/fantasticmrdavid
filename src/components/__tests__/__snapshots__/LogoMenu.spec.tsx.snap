// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component -> LogoMenu should render the LogoMenu UI without throwing an error 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="styles__Container-sc-1fdvout-0 aFnmz"
      >
        <h1
          class="styles__Title-sc-1fdvout-1 cJAooh"
        >
          Fantastic Mr David
        </h1>
        <nav
          class="styles__Nav-sc-1fdvout-3 cwEdZi"
        >
          <div
            class="styles__Container-sc-1w8m3j0-0 dmrGVg"
          >
            <div
              class="styles__Label-sc-1w8m3j0-1 fxEqad"
              target="about"
            >
              About
            </div>
            <div
              class="styles__Content-sc-xptth5-0 cgeHFj"
            >
              <div>
                Meet the captain of the ship.
              </div>
            </div>
          </div>
          <div
            class="styles__Container-sc-1w8m3j0-0 dmrGVg"
          >
            <div
              class="styles__Label-sc-1w8m3j0-1 kLVSUR"
              target="work"
            >
              Latest Work
            </div>
            <div
              class="styles__Content-sc-xptth5-0 cgeHFj"
            >
              <div>
                Some things I've been working on.
              </div>
            </div>
          </div>
          <div
            class="styles__Container-sc-1w8m3j0-0 dmrGVg"
          >
            <div
              class="styles__Label-sc-1w8m3j0-1 oFPLn"
              target="contact"
            >
              Contact
            </div>
            <div
              class="styles__Content-sc-xptth5-0 cgeHFj"
            >
              <div>
                For relaxing times, make it Suntory time.
              </div>
            </div>
          </div>
        </nav>
        <div
          class="styles__Subtitle-sc-1fdvout-4 fnJwRI"
        >
          <h2>
            <div
              class="styles__CogIconWrapper-sc-1fdvout-5 gbVFgv"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
                />
              </svg>
            </div>
            Art + Engineering
          </h2>
          <div
            class="styles__Cursor-sc-cbfjlf-0 OzODf"
          />
        </div>
        <a
          class="styles__Cta-sc-1fdvout-2 gHsBxM"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 320 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
            />
          </svg>
        </a>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="styles__Container-sc-1fdvout-0 aFnmz"
    >
      <h1
        class="styles__Title-sc-1fdvout-1 cJAooh"
      >
        Fantastic Mr David
      </h1>
      <nav
        class="styles__Nav-sc-1fdvout-3 cwEdZi"
      >
        <div
          class="styles__Container-sc-1w8m3j0-0 dmrGVg"
        >
          <div
            class="styles__Label-sc-1w8m3j0-1 fxEqad"
            target="about"
          >
            About
          </div>
          <div
            class="styles__Content-sc-xptth5-0 cgeHFj"
          >
            <div>
              Meet the captain of the ship.
            </div>
          </div>
        </div>
        <div
          class="styles__Container-sc-1w8m3j0-0 dmrGVg"
        >
          <div
            class="styles__Label-sc-1w8m3j0-1 kLVSUR"
            target="work"
          >
            Latest Work
          </div>
          <div
            class="styles__Content-sc-xptth5-0 cgeHFj"
          >
            <div>
              Some things I've been working on.
            </div>
          </div>
        </div>
        <div
          class="styles__Container-sc-1w8m3j0-0 dmrGVg"
        >
          <div
            class="styles__Label-sc-1w8m3j0-1 oFPLn"
            target="contact"
          >
            Contact
          </div>
          <div
            class="styles__Content-sc-xptth5-0 cgeHFj"
          >
            <div>
              For relaxing times, make it Suntory time.
            </div>
          </div>
        </div>
      </nav>
      <div
        class="styles__Subtitle-sc-1fdvout-4 fnJwRI"
      >
        <h2>
          <div
            class="styles__CogIconWrapper-sc-1fdvout-5 gbVFgv"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
              />
            </svg>
          </div>
          Art + Engineering
        </h2>
        <div
          class="styles__Cursor-sc-cbfjlf-0 OzODf"
        />
      </div>
      <a
        class="styles__Cta-sc-1fdvout-2 gHsBxM"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 320 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
          />
        </svg>
      </a>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
